test/test.js

```diff
@@ -93,6 +93,16 @@ describe('stat-mode', function () {
           });
         });
       });
+
+      return test;
+    }).map(function (test) {
```

Comment:
Instead of chaining here with another `.map()` call. instead just add this logic to the above forEach() block. You'll need to add another `describe()` call to keep everything properly nested in the test output.